### PR TITLE
FlxText: bring in line with new Openfl changes.

### PR DESCRIPTION
**Pull request by larsiusprime**

### CODE REVIEW

File: flixel/text/FlxText.hx [hunk 1]
```diff
@@ -532,12 +532,16 @@ class FlxText extends FlxSprite
...
 	}
 	
-	private inline function get_size():Float
+	private inline function get_size():Int
 	{
-		return _defaultFormat.size;
+		#if openfl_legacy
+			return Std.int(_defaultFormat.size);
+		#else
+			return _defaultFormat.size;
```
Comment (Gama11) [inline flixel/text/FlxText.hx]: This code seems to trigger even when compiling with legacy (leading to a compiler error: `Null<Float> should be Int`)

**Comment by larsiusprime** [comment]
I did away with some of the #ifs, just using an Std.int(). Does that get optimized away if the underyling type is already an int?

**Comment by MSGhero** [comment]
@larsiusprime yes in js without analyzer, probably the rest too

**Comment by Gama11** [comment]
Probably not that big of a deal...

The flixel-ui counterpart of this (HaxeFlixel/flixel-ui#107) still uses conditionals btw.

**Pull request merged**